src/style.css

```diff
@@ -1 +1,75 @@
 /* Styles go here */
+ul {
```

Comment:
check out point 1 in the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)